sample-apps/react-native/dogfood/src/components/VideoWrapper.tsx

```diff
@@ -75,6 +77,27 @@ export const VideoWrapper = ({ children }: PropsWithChildren<{}>) => {
             : undefined,
         },
       });
+
+      _videoClient.on('call.rejected', async (event) => {
+        // Workaround needed for the busy tone:
+        // This is because the call was rejected without even starting,
+        // before calling the stop method with busy tone we need to start the call first.
+        InCallManager.start({ media: 'audio' });
+
+        const callCid = event.call_cid;
+        const callId = callCid.split(':')[1];
+        const rejectedCall = _videoClient?.call(event.call.type, callId);
+        await rejectedCall?.getOrCreate();
+
+        const isCalleeBusy =
+          rejectedCall && rejectedCall.isCreatedByMe && event.reason === 'busy';
+
+        if (isCalleeBusy) {
+          InCallManager.stop({ busytone: '_DTMF_' });
+          Alert.alert('Call rejected because user is busy.');
+        }
+      });
```

Comment:
I dont get it, why is this on dogfood? 

This should be on our outgoing call component in SDK